curated_transformers/layers/__init__.py

```diff
@@ -17,7 +18,37 @@
 from .normalization import RMSNorm
 from .transformer import (
     DecoderLayer,
+    EmbeddingsDropouts,
+    EmbeddingsLayerNorms,
     EncoderLayer,
     TransformerDropouts,
+    TransformerEmbeddings,
     TransformerLayerNorms,
 )
+
+__all__ = [
+    "Activation",
+    "AttentionHeads",
+    "AttentionLinearBiases",
+    "AttentionMask",
+    "CacheProtocol",
```

Comment:
We should add this to the API docs as well.

Reply:
I'll make a separate PR, since we should also move this to `models` (it's only used by the model outputs).